jupyter_server/services/contents/filecheckpoints.py

```diff
@@ -2,6 +2,7 @@
 File-based Checkpoints implementations.
 """
 import os
+from os import getcwd
```

Comment:
Rather than introduce a `from os import` statement when `import os` is already used (and gets flagged by the code scanning CI), could we just prefix the single call to `getcwd()` (L51) with `os.` as was the case in other files?

Reply:
Sure. Done.